### ISSUE TITLE
Visualise FT_sensors

### ISSUE THREAD

**Issue by nunoguedelha** [ami-iit/yarp-openmct#126]

**Comment by nunoguedelha** [comment]
Ciao @S-Dafarra . Could you let me know which main config file you are using when activating the FT sensors, or simply which files in https://github.com/ami-iit/robots-configuration/blob/devel_iCubGenova09/iCubGenova09/wrappers/FT you intend to use for activating the FTs?

**Comment by S-Dafarra** [comment]
Our main configuration file is https://github.com/ami-iit/robots-configuration/blob/devel_iCubGenova09/iCubGenova09/icub_wbd.xml

**Comment by nunoguedelha** [comment]
Thanks!

**Comment by nunoguedelha** [comment]
@S-Dafarra , out of curiosity, if I understood correctly from the configuration files, the heel and toe FTs...
- used to be connected respectively to the boards/devices `right_leg-eb11-j0_2-strain` and `right_leg-eb12-j3_5-strain`,
- and are now both connected to the same board/device `right_leg-eb12-j3_5-strain`.

Right?

**Comment by nunoguedelha** [comment]
> @S-Dafarra , out of curiosity, if I understood correctly from the configuration files, the heel and toe FTs...
> 
> * used to be connected respectively to the boards/devices `right_leg-eb11-j0_2-strain` and `right_leg-eb12-j3_5-strain`,
> * and are now both connected to the same board/device `right_leg-eb12-j3_5-strain`.
> 
> Right?

as pr the wrapper in the table and the configuration file https://github.com/ami-iit/robots-configuration/blob/devel_iCubGenova09/iCubGenova09/hardware/FT/right_leg-eb12-j3_5-strain.xml.

**Comment by S-Dafarra** [comment]
Yes, before only one EMS could have handled only one FT. But when we installed some new FTs in the thigh, we ran out of EMS. So we needed to change the FW in order to handle multiple FTs from the same board.

**Comment by nunoguedelha** [comment]
whenever you can, until tomorrow noon, if you could drop here a sample of the Yarp data from one of the ports in the description (e.g. `/icub/left_foot_heel_tiptoe/measures:o`), it would be great.

**Comment by S-Dafarra** [comment]
> whenever you can, until tomorrow noon, if you could drop here a sample of the Yarp data from one of the ports in the description (e.g. `/icub/left_foot_heel_tiptoe/measures:o`), it would be great.

We have some issue with the robot at the moment and I am not able to get those data yet.

**Comment by S-Dafarra** [comment]
> whenever you can, until tomorrow noon, if you could drop here a sample of the Yarp data from one of the ports in the description (e.g. `/icub/left_foot_heel_tiptoe/measures:o`), it would be great.

Here you go:
```
() () () () (((264.0) 1662118197.98573541641) ((264.0) 1662118197.98173546791)) (((-57.952880859375 -101.806640625 8.23974609375 -1.666259765625 0.4852294921875 0.2471923828125) 157041978.0) ((-78.55224609375 -60.05859375 -54.74853515625 -0.30670166015625 1.190185546875 -0.01373291015625) 157037978.0)) () () () ()
() () () () (((264.0) 1662118197.99573731422) ((264.0) 1662118197.99173736572)) (((-57.037353515625 -101.25732421875 6.77490234375 -1.666259765625 0.47607421875 0.2471923828125) 157051980.0) ((-78.55224609375 -59.6923828125 -55.11474609375 -0.2838134765625 1.190185546875 -0.00457763671875) 157047980.0)) () () () ()
() () () () (((264.0) 1662118198.00573444366) ((264.0) 1662118198.00173449516)) (((-57.403564453125 -101.98974609375 7.50732421875 -1.68914794921875 0.47149658203125 0.2471923828125) 157061977.0) ((-78.460693359375 -60.150146484375 -53.8330078125 -0.3021240234375 1.1993408203125 -0.0091552734375) 157057977.0)) () () () ()
() () () () (((264.0) 1662118198.00573444366) ((264.0) 1662118198.01173734665)) (((-57.403564453125 -101.98974609375 7.50732421875 -1.68914794921875 0.47149658203125 0.2471923828125) 157061977.0) ((-78.55224609375 -59.50927734375 -55.2978515625 -0.28839111328125 1.1993408203125 -0.02288818359375) 157067980.0)) () () () ()
() () () () (((264.0) 1662118198.02574038506) ((264.0) 1662118198.02174139023)) (((-57.403564453125 -100.982666015625 7.32421875 -1.67083740234375 0.494384765625 0.25177001953125) 157081983.0) ((-78.643798828125 -59.234619140625 -54.5654296875 -0.28839111328125 1.20849609375 -0.00457763671875) 157077984.0)) () () () ()

```

**Comment by nunoguedelha** [comment]
Corrected the format documentation in the description.
